### PR TITLE
Update converter route responsive design

### PR DESCRIPTION
**Pull request by maryx**

### CODE REVIEW

File: unit_converter/unit_converter/lib/converter_route.dart [hunk 1]
```diff
@@ -286,37 +289,13 @@ class _ConverterRouteState extends State<ConverterRoute> {
...
-              Expanded(
-                child: output,
-              ),
-            ],
-          );
-        }
-      },
+    var converter = Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: <Widget>[
```
Comment (yjbanov) [inline unit_converter/unit_converter/lib/converter_route.dart]: nit: feel free to drop the `<Widget>` for the sake of brevity. Type inference should figure it out for you.

File: unit_converter/unit_converter/lib/converter_route.dart [hunk 2]
```diff
@@ -346,9 +325,26 @@ class _ConverterRouteState extends State<ConverterRoute> {
 //      ],
 //    );
 
+    // Based on the box constraints of our device, figure out how to best
+    // lay out our conversion screen
```
Comment (yjbanov) [inline unit_converter/unit_converter/lib/converter_route.dart]: nit: end sentence with a period.

File: unit_converter/unit_converter/lib/converter_route.dart [hunk 3]
```diff
@@ -346,9 +325,26 @@ class _ConverterRouteState extends State<ConverterRoute> {
...
     return Padding(
       padding: _padding,
-      child: conversionScreen,
+      child: LayoutBuilder(
```
Comment (yjbanov) [inline unit_converter/unit_converter/lib/converter_route.dart]: tip: see also https://docs.flutter.io/flutter/widgets/OrientationBuilder-class.html

**Comment by yjbanov** [comment]
lgtm

File: unit_converter/unit_converter/lib/converter_route.dart (hunk 1 above)
Comment (maryx) [reply unit_converter/unit_converter/lib/converter_route.dart]: Thanks

File: unit_converter/unit_converter/lib/converter_route.dart (hunk 2 above)
Comment (maryx) [reply unit_converter/unit_converter/lib/converter_route.dart]: Done

File: unit_converter/unit_converter/lib/converter_route.dart (hunk 3 above)
Comment (maryx) [reply unit_converter/unit_converter/lib/converter_route.dart]: Makes sense.

**Pull request merged**